runtime/lua/vim/_editor.lua

```diff
@@ -810,6 +810,44 @@ function vim.deprecate(name, alternative, version, plugin, backtrace)
   end
 end
 
+--- Create builtin mappings (incl. menus).
+--- Called once on startup.
+function vim._init_default_mappings()
+  -- mappings
+
+  --@private
+  local function map(mode, lhs, rhs)
+    vim.api.nvim_set_keymap(mode, lhs, rhs, { noremap = true, desc = 'Nvim builtin' })
+  end
+
+  map('n',  'Y',      'y$')
+  -- Use normal! <C-L> to prevent inserting raw <C-L> when using i_<C-O>
+  -- See https://github.com/neovim/neovim/issues/17473
+  map('n',  '<C-L>',  '<Cmd>nohlsearch<Bar>diffupdate<Bar>normal! <C-L><CR>')
+  map('i',  '<C-U>',  '<C-G>u<C-U>')
+  map('i',  '<C-W>',  '<C-G>u<C-W>')
+  map('x',  '*',      'y/\\V<C-R>"<CR>')
+  map('x',  '#',      'y?\\V<C-R>"<CR>')
+
+  -- menus
+
+  -- TODO VimScript, no l10n
+  vim.cmd [[
+    aunmenu *
+    anoremenu PopUp.Undo                    u
+    vnoremenu PopUp.Cut                     "+x
+    vnoremenu PopUp.Copy                    "+y
+    anoremenu PopUp.Paste                   "+gP
+    vnoremenu PopUp.Paste                   "+p
+    vnoremenu PopUp.Delete                  "_x
+    nnoremenu PopUp.Select\ All             ggVG
```

Comment:
todo(?): for many of these, it might be nice to maintain the cursor position and the viewport, so there is no visible jumping-around

Reply:
At least for "Select All" and other Visual-related stuff, it's impossible, AFAIU. The editor really wants to keep the cursor both visible and anchored to one of selection ends, so it may adjust the selected range upon any viewport change.